packages/config/config/devices/0x0345/templates/swidget_template.json

```diff
@@ -0,0 +1,1504 @@
+		// param 54b1
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL ASSOCIATIONS ON EXIT",
+		"description": "If set, send BASIC SET with “Motion Detected Exit SET Value” to “Motion” association group on exit from motion detected state (Vacant)"
+	},
+	"motion_actions_control_host_on_entry": {
+		// param 54b2
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL HOST ON ENTRY",
+		"description": "If set, execute BASIC SET with “Motion Detected Entry SET Value” on host device on entry into motion detected state (Occupied)"
+	},
+	"motion_actions_control_host_on_exit": {
+		// param 54b3
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL HOST ON EXIT",
+		"description": "If set, execute BASIC SET with “Motion Detected Exit SET Value” on host device on exit from motion detected state (Vacant)."
+	},
+	"motion_actions_report_transitions": {
+		// param 54b4
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: NOTIFY TRANSITIONS",
+		"description": "If set, execute BASIC SET with “Motion Detected Exit SET Value” on host device on exit from motion detected state (Vacant)."
+	},
+    "air_quality_level_1_index": {
+		// param 64
+		"label": "Air Quality Level 1 Index",
+		"description": "Used to set the index value representing the upper limit for the 'Clean' air state.",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 500,
+		"unsigned": true,
+		"defaultValue": 100
+
+	},
+    "air_quality_level_2_index": {
+		// param 65
+		"label": "Air Quality Level 2 Index",
+		"description": "Used to set the index value representing the upper limit for the 'Clean' air state.",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 500,
+		"unsigned": true,
+		"defaultValue": 175
+	},
+    "air_quality_level_3_index": {
+		// param 66
+		"label": "Air Quality Level 3 Index",
+		"description": "Used to set the index value representing the upper limit for the 'Moderately Polluted' air state.",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 500,
+		"unsigned": true,
+		"defaultValue": 250
+	},
+	"air_quality_alert_level": {
+		// pram 67
+		"label": "Air Quality Alert Level",
+		"description": "Used to set the air quality level that will trigger Air Quality Alert State actions",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 3,
+		"defaultValue": 2,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Slightly Polluted",
+				"value": 1
+			},
+			{
+				"label": "Moderately Polluted",
+				"value": 2
+			},
+			{
+				"label": "Extremely Polluted",
+				"value": 3
+			}
+		]
+	},
+	"air_quality_alert_delay": {
+		// param 68
+		"label": "Air Quality Alert Delay",
+		"description": "Delay after detection of a level change before acting on change of air quality state",
```

Comment:
Format this and the following params similar to the previous one. Descriptions all seem unnecessary.

Reply:
These are the parameter names per the manufacturers zwave documentation. I would prefer to keep them in-sync.